src/V3Ast.h

```diff
@@ -2458,36 +2458,8 @@ inline AstNode* VNVisitor::iterateSubtreeReturnEdits(AstNode* nodep) {
     return nodep->iterateSubtreeReturnEdits(*this);
 }
 
-// ######################################################################
-//  Standard defines for all AstNode subclasses
-
-#define ASTNODE_BASE_FUNCS(name) \
-    virtual ~Ast##name() override = default; \
-    static Ast##name* cloneTreeNull(Ast##name* nodep, bool cloneNextLink) { \
-        return nodep ? nodep->cloneTree(cloneNextLink) : nullptr; \
-    } \
-    Ast##name* cloneTree(bool cloneNext) { \
-        return static_cast<Ast##name*>(AstNode::cloneTree(cloneNext)); \
-    } \
```

Comment:
We should put comments here showing what's generated by astnodegen, to help new coders have an idea what's going on.

Reply:
I will add some comments along the lines of "astgen generates some standard methods of the form blah, but read the generated file to see what exactly", enumerating everything precisely is a bit tedious as there can be quite some variation.